templates/taxi/car_detail.html

```diff
@@ -9,4 +9,7 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
+  <hr>
+    <a href="{% url 'taxi:car-update' pk=car.id %}">Update</a>
+    <a href="{% url 'taxi:car-delete' pk=car.id %}">Delete</a>
```

Comment:
According to the checklist, you should add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on.